wpengine/faustjs
# Fix WP Plugin Versioning

## Description

This PR introduces two fixes:

* Moves the screenshots and FAQs sections above the changelog section in the plugin's `readme.txt` so they are not overwritten by the `versionPlugin` script.
* Renames the `FaustWP` header in the plugin's `CHANGELOG.md` to `Faust` to fix the rogue `WP` showing up in the plugin's readme when versioning (i.e. `=== Changelog ===WP`)

## Testing

1. Switch to this branch
2. Add `WP` After `=== Changelog ===` in the plugin's `readme.txt`
3. From the monorepo root, run `node scripts/versionPlugin.js`
4. Notice that the line has been fixed
